### PR TITLE
Update NEWS, README and website for 1.7.0

### PR DESCRIPTION
**Pull request by ciyongch**
Update NEWS README and website for 1.7.0 release.

@szha @TaoLv @leezu

### CODE REVIEW

**Comment by TaoLv** [comment]
It's strange. I remember I changed the download table to use mirror links in https://github.com/apache/incubator-mxnet/pull/16501.

**Review by leezu — CHANGES_REQUESTED** [review]
Must use archive.apache.org for all but the latest release. See https://infra.apache.org/release-publishing.html

**Comment by ciyongch** [comment]
> It's strange. I remember I changed the download table to use mirror links in #16501.

They're changed by #18487

**Comment by ciyongch** [comment]
> Must use archive.apache.org for all but the latest release. See https://infra.apache.org/release-publishing.html

Sure, I will update the download link according to this spec. Thanks!

**Comment by ciyongch** [comment]
Hi @leezu , I've updated the download link, please help to take a review.

File: NEWS.md [hunk 1]
```diff
@@ -18,120 +18,161 @@
...
 - [MXNet Change Log](#mxnet-change-log)
-  * [1.6.0](#160)
-    + [Deprecation of Python 2](#deprecation-of-python-2)
-    + [New Features](#new-features)
-      - [NumPy compatible interface and using TVM to generate operators](#numpy-compatible-interface-and-using-tvm-to-generate-operators)
-      - [Graph optimizations](#graph-optimizations)
+  - [1.7.0](#170)
+    - [New features](#new-features)
+      - [MXNet Extensions: custom operators, partitioning, and graph passes](#mxnet-extensions-custom-operators-partitioning-and-graph-passes)
+      - [OpPerf utility enabled in the binary distribution](#opperf-utility-enabled-in-the-binary-distribution)
```
Comment (ChaiBapchya) [inline NEWS.md:24]: which PR did this?

Comment (ciyongch) [reply NEWS.md:24]: Hi @ChaiBapchya , probably it's not drived by any PR in the current scope, but included in the binary release process. cc @szha to see if any comments.
When trying to install mxnet release (1.7.0) wheel via pip, the benchmark/opperf is also installed now. And the original proposal from you is listed in [1.7.0 roadmap](https://github.com/apache/incubator-mxnet/issues/16864#issuecomment-585461851)

**Comment by ciyongch** [comment]
Hi @leezu @szha @marcoabreu @ChaiBapchya @TaoLv @pengzhao-intel , please help to take a review and comment if anything needs to be updated, thanks!

**Review by marcoabreu — APPROVED** [review]

File: docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md [hunk 2]
```diff
@@ -9,17 +9,28 @@ page](https://mxnet.apache.org/get_started/download).
...
 
-<div class="v1-6-0">
+<div class="v1-7-0">
 {% highlight bash %}
 pip install mxnet
 {% endhighlight %}
 
+Start from 1.7.0 release, MKL-DNN is enabled in pip packages by default. Which are
+optimized for Intel hardware. You can find performance numbers
+in the <a href="https://mxnet.io/api/faq/perf#intel-cpu">MXNet tuning guide</a>.
```
Comment (leezu) [inline docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md:19]: Link to mxnet.apache.org/

File: docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md [hunk 3]
```diff
@@ -9,17 +9,28 @@ page](https://mxnet.apache.org/get_started/download).
 
 Run the following command:
 
-<div class="v1-6-0">
+<div class="v1-7-0">
 {% highlight bash %}
 pip install mxnet
 {% endhighlight %}
 
+Start from 1.7.0 release, MKL-DNN is enabled in pip packages by default. Which are
```
Comment (leezu) [inline docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md:17]: oneDNN?

File: docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md [hunk 4]
```diff
@@ -9,17 +9,28 @@ page](https://mxnet.apache.org/get_started/download).
...
+optimized for Intel hardware. You can find performance numbers
```
Comment (leezu) [inline docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md:18]: Elaborate on the "optimization? For example "oneAPI Deep Neural Network Library (oneDNN) is an open-source cross-platform performance library of basic building blocks for deep learning applications. The library is optimized for Intel Architecture Processors, Intel Processor Graphics and Xe architecture-based Graphics. Support for other architectures such as Arm* 64-bit Architecture (AArch64) and OpenPOWER* Power ISA (PPC64) is experimental." (from the oneDNN repo).

Please mention that the `mxnet-native` release is available which does come without oneDNN

Comment (ciyongch) [reply docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md:18]: Thank you for the valuable inputs, leezu, I will update the description here.

Comment (ciyongch) [reply docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md:18]: BTW, I didn't find available `mxnet-native` pip wheel via `pip install mxnet-native`, is it still WIP or using the different name?

File: docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md (hunk 2 above)
Comment (ciyongch) [reply docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md]: The perf link under mxnet.apache.org will contain the version number like "https://mxnet.apache.org/versions/1.6/api/faq/perf.html#intel-cpu". Is there any basic link without version number which can redirect to the latest version of the page? 
Please check my latest update.

**Comment by ciyongch** [comment]
@mxnet-bot run ci [windows-gpu]

File: docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md (hunk 4 above)
Comment (szha) [reply docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md]: yes it's WIP.

**Pull request merged**